app/views/groups/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<header class="bg-primary p-2 d-flex justify-content-around align-items-center">
+  <div>
+    <i class="bi bi-wallet-fill fs-1 text-light"></i>
+  </div>
+  <div>
+    <h1 class="text-center text-light"> All Groups </h1>
+  </div>
+  <div>
```

Comment:
- [ ] For each category, the user should be able to see their name, icon and the total amount of all the transactions that belongs to that category. please add name, it is missing: 
<img width="410" alt="Screenshot 2022-07-01 at 13 25 06" src="https://user-images.githubusercontent.com/77038610/176866908-fa8eba6a-0436-47f0-995e-4d81b667a257.png">

Reply:
Hello @ShoiraTa is this really compulsory? Adding the user name attributes to the created groups

Reply:
Hello, while reviewing your project I had to follow the provided guidelines, and in the guidelines this requirement is compulsory, sorry 😢 
But this is not difficult to implement since you already have the relationships, something like @group.user.name should be printing the user name, if you have no user name you can use user email @group.user.email  , hope this was helpful ))